File: src/main/java/br/com/dusty/dkits/listener/mechanics/WeatherChangeListener.java
package br.com.dusty.dkits.listener.mechanics;

import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.weather.WeatherChangeEvent;

public class WeatherChangeListener implements Listener {
	
	@EventHandler
	public static void onWeatherChange(WeatherChangeEvent event) {
		event.setCancelled(true);
	}
}

File: src/main/kotlin/br/com/dusty/dkits/listener/mechanics/WeatherChangeListener.kt
package br.com.dusty.dkits.listener.mechanics

import org.bukkit.event.EventHandler
import org.bukkit.event.EventPriority
import org.bukkit.event.Listener
import org.bukkit.event.weather.WeatherChangeEvent

object WeatherChangeListener: Listener {

	@EventHandler
	fun onWeatherChange(event: WeatherChangeEvent) {
		event.isCancelled = true
	}
}
